main.go

```diff
@@ -0,0 +1,221 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/go-utils/sliceutil"
+	"github.com/bitrise-tools/go-android/gradle"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+	"github.com/bitrise-tools/go-steputils/tools"
+)
+
+const (
+	apkEnvKey          = "BITRISE_APK_PATH"
+	apkListEnvKey      = "BITRISE_APK_PATH_LIST"
+	mappingFileEnvKey  = "BITRISE_MAPPING_PATH"
+	mappingFilePattern = "*build/*/mapping.txt"
+)
+
+// Config ...
+type Config struct {
+	ProjectLocation string `env:"project_location,required"`
+	APKPathPattern  string `env:"apk_path_pattern"`
+	Variant         string `env:"variant"`
+	Module          string `env:"module"`
+}
+
+func failf(f string, args ...interface{}) {
+	log.Errorf(f, args...)
+	os.Exit(1)
+}
+
+func main() {
+	var config Config
```

Comment:
`Config` type shouldn't be exported, while for `config` var a shorter name, like `cfg` is more conventional.